share/extender/build_input.yml

```diff
@@ -251,15 +251,18 @@ platforms:
     env:
         PROGUARD:               "{{env.ANDROID_PROGUARD}}"
         LIBRARYJAR:             "{{env.ANDROID_LIBRARYJAR}}"
-        NDK_PATH:               "{{env.ANDROID_NDK20_PATH}}"
-        SYSROOT:                "{{env.ANDROID_NDK20_SYSROOT}}"
-        NDK_VERSION:            "{{env.ANDROID_NDK20_API_VERSION}}"
-        STL_LIB:                "{{env.ANDROID_NDK20_STL_LIB}}"
+        NDK_PATH:               "{{env.ANDROID_NDK25_PATH}}"
+        PATH:                   "{{env.ANDROID_NDK25_BIN_PATH}}:{{env.PATH}}"
+        SYSROOT:                "{{env.ANDROID_NDK25_SYSROOT}}"
+        NDK_VERSION:            "{{env.ANDROID_NDK25_API_VERSION}}"
         MANIFEST_MERGE_TOOL:    "{{env.MANIFEST_MERGE_TOOL}}"
+        PATH:                   "{{env.ANDROID_NDK25_BIN_PATH}}:{{env.PATH}}"
+        NDK_LD:                 "{{env.ANDROID_NDK25_BIN_PATH}}/ld.lld"
```

Comment:
To make sure the compiler uses the linker I expected I use absolute path for `-fuse-ld`.
It’s not obvious from the official clang documentation that it is possible to use absolute path, but here is [a mention](https://patchwork.kernel.org/project/linux-kbuild/patch/20190211193008.24101-3-ndesaulniers@google.com/) about this possibility.